generators/setup-docker/templates/fabric-docker/commands-generated.sh

```diff
@@ -16,23 +16,36 @@ function installChaincodes() {
   <% } else { -%>
     <% chaincodes.forEach(function(chaincode) { -%>
       <%- include('commands-generated-node-build.sh.ejs', {chaincode: chaincode}); -%>
-      <% chaincode.channel.orgs.forEach(function (org) {
-           org.peers.forEach(function (peer) {
-      %>
-      printHeadline "Installing '<%= chaincode.name %>' on <%= chaincode.channel.name %>/<%= org.name %>/<%= peer.name %>" "U1F60E"
-      <% if(!networkSettings.tls) { -%>
-      chaincodeInstall "cli.<%= org.domain %>" "<%= peer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "<%= chaincode.version %>" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>"
-      <% } else { -%>
-      chaincodeInstallTls "cli.<%= org.domain %>" "<%= peer.fullAddress %>" "<%= chaincode.channel.name %>" "<%= chaincode.name %>" "<%= chaincode.version %>" "<%= chaincode.lang %>" "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" "<%= rootOrg.ordererHead.fullAddress %>" "crypto/orderer-tlscacerts/tlsca.<%= rootOrg.domain %>-cert.pem"
-      <% } -%>
-      <% })}) -%>
+      <% chaincode.channel.orgs.forEach(function (org) { -%>
+        <% org.peers.forEach(function (peer) { -%>
+          printHeadline "Installing '<%= chaincode.name %>' on <%= chaincode.channel.name %>/<%= org.name %>/<%= peer.name %>" "U1F60E"
+          chaincodeInstall \
```

Comment:
hm... not sure about this one. `commands-generated.sh` tends to be big file. We also can install multiple chaincodes on multiple channels and peers. I would keep it as one liner as it's easier to reason about whole network setup that way.